src/core/Utilities/ImageHelper.ts

```diff
@@ -9,29 +9,29 @@ import { Desktop } from '../Desktop'
 /**
  * Provides helper functions to handle image pixel data.
  */
-export class ImageHelper {
+export const ImageHelper = {
   /**
    * puts image on canvas using the parent canvas ctx.
    * @param parent parent desktop with CTX for canvas
    * @param x x loc
    * @param y y loc
    */
-  static putImage (parent: Desktop, x: number, y: number): any {
+  putImage (parent: Desktop, x: number, y: number): any {
     if (parent.holding) return
 
     const xx = ImageHelper.arotX(parent, x, y)
     y = ImageHelper.arotY(parent, x, y)
     x = xx
     parent.canvasCtx.putImageData(parent.spare, x, y)
-  }
+  },
 
   /**
    *
    * @param parent parent desktop
    * @param value pixel value at ptr
    * @param ptr ptr into the image pixel data
    */
-  static setPixel (parent: Desktop, value: any, ptr: number): any {
+  setPixel (parent: Desktop, value: any, ptr: number): any {
```

Comment:
is value a number? and return type should be void.

Reply:
updated